src/Microsoft.AspNet.Server.Kestrel/App.config

```diff
@@ -0,0 +1,4 @@
+﻿<configuration>
+  <dllmap dll="libuv.dll" target="libuv.so.1" os="linux"/>
+  <dllmap dll="libuv.dll" target="native/darwin/universal/libuv.dylib" os="osx"/>
```

Comment:
Does the file need to get included in the NuGet package? Does that happen automatically in this build system? (Sorry for the n00b questions on this!)